packages/design-system-tokens/src/css/__snapshots__/css-snapshots.test.ts.snap

```diff
@@ -416,285 +416,285 @@ exports[`CSS snapshots variables in cmsgov-theme.css match snapshot 1`] = `
+--button-solid-dark-alt__color: var(--color-base);
+--button-solid-dark__background-color--active: var(--color-gray-lighter);
+--button-solid-dark__background-color--disabled: var(--color-gray-darker);
+--button-solid-dark__background-color--hover: var(--color-white);
+--button-solid-dark__background-color: var(--color-gray-lightest);
+--button-solid-dark__border-color--active: var(--color-gray-lighter);
+--button-solid-dark__border-color--disabled: var(--color-gray-darker);
+--button-solid-dark__border-color--hover: var(--color-white);
+--button-solid-dark__border-color: var(--color-gray-lightest);
+--button-solid-dark__color--active: var(--color-base);
+--button-solid-dark__color--disabled: var(--color-base);
+--button-solid-dark__color--hover: var(--color-base);
+--button-solid-dark__color: var(--color-base);
+--button-solid__background-color--active: var(--color-primary-darkest);
+--button-solid__background-color--disabled: var(--color-gray-lighter);
+--button-solid__background-color--hover: var(--color-primary-darker);
+--button-solid__background-color: var(--color-primary);
+--button-solid__border-color--active: var(--color-primary-darkest);
+--button-solid__border-color--disabled: var(--color-gray-lighter);
+--button-solid__border-color--hover: var(--color-primary-darker);
+--button-solid__border-color: var(--color-primary);
+--button-solid__color--active: var(--color-white);
+--button-solid__color--disabled: var(--color-gray-darker);
+--button-solid__color--hover: var(--color-white);
+--button-solid__color: var(--color-white);
+--button__background-color--active: var(--color-gray-lighter);
+--button__background-color--disabled: var(--color-transparent);
+--button__background-color--hover: var(--color-gray-lightest);
+--button__background-color: var(--color-transparent);
+--button__border-color--active: var(--color-primary-darkest);
+--button__border-color--disabled: var(--color-gray-lighter);
+--button__border-color--hover: var(--color-primary-darker);
+--button__border-color: var(--color-primary);
+--button__border-radius: var(--radius-default);
 --button__border-width: 1px;
---button__color--active: #00395e;
---button__color--disabled: #d9d9d9;
---button__color--hover: #004f84;
---button__color: #0071bc;
---choice-label__color--disabled--inverse: #e9ecf1;
---choice-label__color--disabled: #5a5a5a;
+--button__color--active: var(--color-primary-darkest);
+--button__color--disabled: var(--color-gray-lighter);
+--button__color--hover: var(--color-primary-darker);
+--button__color: var(--color-primary);
+--choice-label__color--disabled--inverse: var(--color-muted-inverse);
+--choice-label__color--disabled: var(--color-muted);
 --choice-label__top-offset: 0.25rem;
 --choice-wrapper__gap--small: 0.5rem;
 --choice-wrapper__gap: 0.5rem;
---choice__background-color--checked: #0071bc;
+--choice__background-color--checked: var(--color-primary);
 --choice__background-color--disabled--inverse: #e9ecf126;
```

Comment:
this hex looks weird. i assume there's an alpha applied?

Reply:
Yeah, the extra two digits are the alpha channel